R/bcdc_browse.R

```diff
@@ -85,3 +85,34 @@ bcdc_browse <- function(query = NULL, browser = getOption("browser"),
 
 
 
+#' Generate a bibentry from a Data Catalogue Record
+#'
+#' Generate a bibentry object for use .bib file.
+#' for more details
+#'
+#' @inheritParams bcdc_get_record
+#' @export
+#'
+#' @seealso \link{utils::bibentry}
+#'
+#' @examples
+#'
+#' try(
+#' cite <- bcdc_get_data_citation("76b1b7a3-2112-4444-857a-afccf7b20da8")
+#'
+#' print(cite, stype = "citation")
+#' )
+bcdc_get_data_citation <- function(id) {
+  rec <- bcdc_get_record(id)
```

Comment:
Do we want to make this a generic so we have a `"character"` and `"bcdc_record"` method?

Reply:
Yes absolutely. Great idea.